package.json

```diff
@@ -31,11 +32,13 @@
     "node": ">=0.10.0"
   },
   "devDependencies": {
+    "coveralls": "2.12.0",
     "eslint": "2.12.0",
     "eslint-config-silvermine": "0.9.1",
     "grunt": "1.0.1",
     "grunt-cli": "1.2.0",
     "grunt-eslint": "19.0.0",
-    "mocha": "2.5.3"
+    "mocha": "2.5.3",
+    "nyc": "10.1.2"
```

Comment:
We have a standardized way of measuring coverage on all our projects, and this is not it. Please use the standardized way.